### ISSUE TITLE
Warning when prompts needs to be truncated

### ISSUE THREAD

**Issue by jgranie** [huggingface/swift-coreml-diffusers#53]
Hi,
I understood the previous issue I've posted only when I tried to generate image with command line ml-stable-diffusion. The prompts are truncated and images generated are consistent with what I get in A1111.
Could it be possible to display a warning to avoid any confusion ?
Thanks.

**Comment by pcuenca** [comment]
Hi @jgranie! Sorry, I'm not sure I follow. Would you like a warning when the prompt is too long and has to be truncated?

**Comment by dolmere** [comment]
I'm guessing either something like a twitter style character count down or changing text color to warn of impending or active truncation?

**Comment by jgranie** [comment]
Yes, anything visual that tells us that the prompt will be (or bas been) truncated (before or after generation). Even a message popup can do the job . Thanks for your reply.
